acquire/outputs/zip.py

```diff
@@ -0,0 +1,112 @@
+import io
```

Comment:
```suggestion
import io
import shutil
```
This import is needed for suggestion below.